core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/request/SdxDiskUpdateRequest.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request;
+
+import java.util.List;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import com.sequenceiq.cloudbreak.cloud.model.VolumeSetAttributes;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel
+public class SdxDiskUpdateRequest {
+
+    @NotEmpty
+    private String cloudPlatform;
+
+    @NotNull
+    private List<VolumeSetAttributes.Volume> volumesToUpdate;
```

Comment:
please create a sdx specific volume request type instead of using the general `VolumeSetAttributes.Volume`

Reply:
done. i am reusing the Volume class in cloud model. this will also have to be done for datahub, so not creating a new DL specific class